test/fluree/db/query/misc_queries_test.clj

```diff
@@ -34,28 +36,68 @@
 (deftest ^:integration result-formatting
   (let [conn   (test-utils/create-conn)
         ledger @(fluree/create conn "query-context" {:defaultContext ["" {:ex "http://example.org/ns/"}]})
-        db     @(test-utils/transact ledger [{:id   :ex/dan
-                                              :ex/x 1}])]
+        db     @(fluree/stage (fluree/db ledger) [{:id :ex/dan :ex/x 1}])]
+
+    @(fluree/commit! ledger db)
+
+    (is (= [{:id   :ex/dan
+             :ex/x 1}]
+           @(fluree/query db {:where  [["?s" :id :ex/dan]]
+                              :select {"?s" [:*]}}))
+        "default context")
     (is (= [{:id    :foo/dan
              :foo/x 1}]
            @(fluree/query db {"@context" ["" {:foo "http://example.org/ns/"}]
-                              :where     [['?s :id :foo/dan]]
-                              :select    {'?s [:*]}}))
+                              :where     [["?s" :id :foo/dan]]
```

Comment:
Why the switch to strings?

Reply:
It's kind of a silly reason - when I use `lispy-tab` to indent nested forms in emacs it for some reason transforms quoted symbols, i.e. `'?s` becomes `'115`. I don't know why it does that, but it doesn't do that to strings, so I tend to prefer that syntax while I'm working on a code snippet that I'm indenting often.